### PR TITLE
combine profile and password

### PR DESCRIPTION
**Pull request by YY-coding**
## Changelog
- I changed the update password heading to account setting
- I put the logic from edit profile to account setting where has the update password logic

## Link to issue:  
https://github.com/MidwestAccessCoalition/lockbox_rails/issues/589

## Steps for QA/Special Notes:
- N/A

## Relevant Screenshots: 
When you clicked setting, you will see this
Screen Shot 2021-05-16 at 6 41 15 PM

when you click account setting, you will see this
Screen Shot 2021-05-16 at 6 40 34 PM

## Are you ready for review?:

- [ ] Added relevant tests
- [x] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [x] Added relevant screenshots

### CODE REVIEW

**Comment by jasonwynkoop86** [comment]
Here is my take on the account screen. 
Changes I made. 
Delete edit password button on the right nav. 

Change edit profile to edit account settings. 

Combine account info screens for Change password, name, gmt, and address info.

image

**Comment by jel535s** [comment]
@YY-coding FYI: @jasonwynkoop86 added some designs to #585 that might affect your work here. He also noted that it might make sense to combine this PR with the work spec'd out on #586 . Let me know what you think- is it easier to build that request in to this PR, or would you rather proceed with the work you've already started here and then pick up #586 after?

**Comment by misternu** [comment]
Screen Shot 2021-05-31 at 9 31 56 AM
I found an issue where if there is an error it is displayed on both forms. Theres some other stuff we should try to work out about upcoming work for this page that is currently being designed/redesigned. Will mention during stand and we can probably pair on it and talk to the appropriate people

**Comment by BintLopez** [comment]
Discussed issue with duplicate error messages and decided to simplify by only having one error message div at the top of the page. Note -- fields with error should still have a red border

**Comment by BintLopez** [comment]
Also discussed the ability to change an address. Since the address is on the LockboxPartner (soon to be renamed Organization) instead of on the User, it should not be editable on this page. cc @jasonwynkoop86

**Comment by YY-coding** [comment]
I can only put the error message in the middle. If I put the message on the top, after submitting the change, this error appears
Screen Shot 2021-06-06 at 7 01 50 PM

with the error message in the middle, after refreshing the page, the following error appears
Screen Shot 2021-06-06 at 7 51 14 PM

This is how the page looks like form is not validated
Screen Shot 2021-06-06 at 7 00 19 PM

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -1,36 +1,69 @@
...
+        <h2>
+          <%= @user.errors.count %> error(s) prohibited this user from being saved:
+        </h2>
+        <ul>
+          <% @user.errors.full_messages.each do |error_message| %>
+            <li><%= error_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
```
Comment (BintLopez) [inline app/views/devise/registrations/edit.html.erb:47]: I think instead what we can do is delete lines 36-47 and re-insert `<%= render "devise/shared/error_messages", resource: resource %>` between lines 2 and 3.

Comment (YY-coding) [reply app/views/devise/registrations/edit.html.erb]: @BintLopez I just did what you suggested. The form validation message is on the top of the page now as you can see in the image below. 
Screen Shot 2021-06-13 at 1 17 17 PM

But if I click the refresh arrow while the validation message is on page, it gives me a route error as you can see on the image below

Screen Shot 2021-06-13 at 1 17 40 PM

I also try this in the trunk, it has the same issues. I hope it makes sense.

**Comment by misternu** [comment]
The described issue is a part of normal rails/devise behavior and is in current trunk, merging this as is with a possible upcoming cleanup PR

**Review by misternu — APPROVED** [review]
🦉

**Pull request merged**